std_msgs/Header header
sensor_msgs/Image image
sensor_msgs/Image homographic_transform
uint8 piece_index
geometry_msgs/Point[] transformed_points
geometry_msgs/Point[] piece_surf_key_points
geometry_msgs/Point[] template_surf_key_points
